server/src/main/java/org/opensearch/common/cache/stats/CacheStatsDimension.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.cache.stats;
+
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.common.io.stream.Writeable;
+
+import java.io.IOException;
+import java.util.Objects;
+
+public class CacheStatsDimension implements Writeable {
+    // Values for tier dimensions, that are reused across CacheStats implementations
+    public static final String TIER_DIMENSION_NAME = "tier";
```

Comment:
I thought it would be a reasonable place to have this value for tiers which need it. Maybe TieredSpilloverCache would be a better place? But some places in core (OpenSearchOnHeapCache) need this value as well, and the TSC lives in a module.